src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
+
+	browser.pause(2000);
+
+	const cookieControl = $("body #ccc");
+		if (cookieControl.isVisible("button=Accept all cookies")) {
```

Comment:
I know we used this selector in CKS, but in hindsight, this is maybe a bit fragile as it's relying on the text, maybe `#ccc-recommended-settings` or `.ccc-accept-button` is better?

Reply:
opted to use .ccc-accept-button which worked as expected.